tenets/codelingo/go/declare_empty_slice/codelingo.yaml

```diff
@@ -0,0 +1,22 @@
+tenets:
+  - name: declare_empty_slice
+    flows:
+      codelingo/review:
+        comment: When declaring an empty slice, prefer `var t []string` over `t := []string{}`.
+    query: |
+      import (
+        codelingo/ast/go
+      )
+
+      @review comment
+      go.assign_stmt(depth = any):
+        go.rhs:
+          go.composite_lit:
+            go.array_type:
+              go.ident
+
+              exclude:
+                go.basic_lit
```

Comment:
Do you need this? Shouldn't excluding `go.elts` be enough? Even a slice with a single element has a `go.elts` fact.

Reply:
You're right I'll take it out.